docs/tutorials/custom-commit-notifications.md

```diff
@@ -0,0 +1,124 @@
+Things happen all the time in your development workflow. Commits are pushed,
+builds are started, they fail or succeed. Team members, or even users, open and
+comment issues everyday.
+
+All that flow of information can become overwhelming. With Atomist, you can
+attach behavior to these events and automate all sorts of near real-time
+responses.
+
+These tutorials show you how to rely on Atomist to automate those scenarios.
+
+!!! tip "All you need is Rug"
+    These tutorials aim at showing you the code that you could write to
+    automate event reactions. They assume you have a [Rug project][ugpj]. You
+    can easily add [Rug event handler][rugev] to that project via the public 
+    [AddTypeScriptEventHandler Rug editor][rugeditor] provided by Atomist.
```

Comment:
"These tutorials focus on the code that you would write to
automate event reactions. They assume you have a [Rug project][ugpj] to add these files to. You
can easily add [Rug event handlers][rugev] to your project via the public 
[AddTypeScriptEventHandler Rug editor][rugeditor] provided by Atomist."